### ISSUE TITLE
[ARIA] Ship ARIA reflection (non-idref)

### ISSUE THREAD

**Issue by Rumyra** [mdn/content#29311]
## Acceptance Criteria

- [x] The listed features are documented sufficiently on MDN
- [x] BCD is updated
- [N/A] Interactive example and data repos are updated if appropriate
- [x] The content has been reviewed as needed

### For folks helping with Firefox related documentation

- [x] Set bugs to `dev-doc-complete`
- [x] Add entry to Firefox release notes if feature is enabled in release
- [x] Add entry to Firefox experimental features page if feature is not yet enabled in release

## Related Gecko bugs

https://bugzilla.mozilla.org/show_bug.cgi?id=1785412

## Other

- [x] Check content open issues to see if any pertain to the subject matter. If there are any that can be closed because of the work, do so. If there are any that can be fixed relatively quickly because of the knowledge from completing this issue and you have time, feel free to go ahead and fix them.
- [x] Check if glossary updates are required for the feature you're documenting - whether an existing term needs to be updated or a new term should be added.
- [x] Check if BCD update means that content pages need to have experimental markup removed or deprecated markup added (front matter tags and macros).

**Comment by zfox23** [comment]
## Tasks
- [ ] Update BCD to show FF 119 support for [`ariaSelected`](https://developer.mozilla.org/en-US/docs/Web/API/Element/ariaSelected)
- [ ] Update BCD on [ElementInternals#instance_properties_included_from_aria](https://developer.mozilla.org/en-US/docs/Web/API/ElementInternals#instance_properties_included_from_aria)
    - [ ] Figure out what actually needs to be updated first...
- [ ] Update BCD on [Element#instance_properties_included_from_aria](https://developer.mozilla.org/en-US/docs/Web/API/Element#instance_properties_included_from_aria)
    - [ ] Figure out what actually needs to be updated first...
- [ ] Update [Experimental_features#reflecting_aria_attributes](https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Experimental_features#reflecting_aria_attributes)
    - [ ] Figure out what needs to be updated. I _think_ I need to show that, in DE, Beta, and Release 119, this preference is enabled by default. Is that right?
- [ ] Update the FX119 release notes to show that ARIA Reflection is now supported, and briefly explain what that means.
